src/test/groovy/com/jakuszko/hexagonal/user/domain/UserFacadeSpecification.groovy

```diff
@@ -30,7 +32,8 @@ class UserFacadeSpecification extends UnitTest {
 
     def "should get user"() {
         given:
-        User user = getUserFromRepo()
+        userRepository.withUser()
```

Comment:
Jest lepiej ale nadal czytelniej będzie jeśli dodasz w tej klasie metodę np createSampleUser() i w given wywołasz:
```
given:
def user = createSampleUser()
userRepo.save(user)
```

Możesz też to wyodrębnić do metody jako jeden krok:
```
given:
def user = aUserExists()
```
gdzie aUserExists to coś w stylu:
```
def aUserExists(){
   def user = createSampleUser()
   return userRepo.save(user)
   }
```